models/netsuite2/src_netsuite2.yml

```diff
@@ -2,246 +2,294 @@ version: 2
           - name: receivablesaccount
-            description: The default receiveables account identifier of the customer.
-          - name: _fivetran_deleted
-            description: Boolean created by Fivetran to indicate whether the record has been deleted.
+            description: Reference to the associated receivables account.
+          - name: currency
+            description: "{{ doc('currency_id') }}"
+          - name: firstorderdate
+            description: Timestamp of when the first order was created.
 
       - name: department
-        description: Table detailing all departments set up in Netsuite.
+        identifier: "{{ var('netsuite2_department_identifier', 'department') }}"
+        description: "{{ doc('departments_table') }}"
         columns:
-          - name: identifier
-            description: The unique identifier of the department.
-          - name: externalid
-            description: The unique external identifier of the department.
-          - name: full_name
-            description: The department's full name.
-          - name: isinactive
-            description: Value indicating whether the class is inactive (either yes or no).
-          - name: name
-            description: The name of the department.
+          - name: _fivetran_synced
+            description: "{{ doc('_fivetran_synced') }}"
+          - name: id
+            description: "{{ doc('department_id') }}"
           - name: parent
-            description: The parent department identifier of the department.
+            description: Reference to the parent department.
+          - name: name
+            description: Name of the department.
+          - name: fullname
+            description: Full name of the department.
           - name: subsidiary
-            description: The subsidiary identifier of the department.
+            description: "{{ doc('subsidiary_id') }}"
+          - name: isinactive
+            description: Boolean indicating if the department is active.
           - name: _fivetran_deleted
-            description: Boolean created by Fivetran to indicate whether the record has been deleted.
+            description: "{{ doc('_fivetran_deleted') }}"
 
       - name: entity
-        description: Table detailing all entities in Netsuite.
+        identifier: "{{ var('netsuite2_entity_identifier', 'entity') }}"
+        description: "{{ doc('entities_table') }}"
         columns:
+          - name: id
+            description: Th "{{ doc('entity_id') }}"e unique identifier of the entity.
           - name: contact
             description: The unique identifier of the contact associated with the entity.
           - name: customer
-            description: The unique identifier of the customer associated with the entity.
+            description: The  "{{ doc('customer_id') }}"unique identifier of the customer associated with the entity.
```

Comment:
should it just be `"{{ doc('customer_id') }}"`?

Reply:
Updated